integr8ly/integreatly-operator
# Adds a new test suite for the managed 3scale install type

# Description

Adds a new test suite for the managed 3scale install type

https://issues.redhat.com/browse/MGDAPI-6

## Verification steps

1) install the operator using the managed 3scale install type
 1.1) Export the env variable: `export INSTALLATION_TYPE=managed-3scale`
 1.2) Install the operator locally 
2) Run the functional tests `make test/functional` 
3) Verify all the tests in the managed 3scale install type test suite ran
